homeassistant/components/automation/__init__.py

```diff
@@ -286,6 +287,11 @@ def is_on(self) -> bool:
         """
         if skip_condition or self._cond_func(variables):
             self.async_set_context(context)
+            self.hass.bus.async_fire(EVENT_AUTOMATION_TRIGGER, {
+                ATTR_NAME: self._name,
+                ATTR_DOMAIN: DOMAIN,
```

Comment:
No need to include domain, we know the domain.

Not sure if name should be included either? that seems something that can be extracted when we create logbook entries (more info how I want to change that here https://github.com/home-assistant/architecture/issues/110)

Reply:
I can remove domain, but the way logbook currently works, it looks up the name for the majority of events from the data.  When https://github.com/home-assistant/architecture/issues/110 is implemented, logbook will need to be re-worked anyway, so it can be removed at that time.

Reply:
We already hardcode the domain for `EVENT_HOMEASSISTANT_START`, we should do the same in `logbook.py` for `EVENT_AUTOMATION_TRIGGER`